test_runner/fixtures/neon_fixtures.py

```diff
@@ -3109,3 +3109,18 @@ def last_flush_lsn_upload(
     ps_http.timeline_checkpoint(tenant_id, timeline_id)
     wait_for_upload(ps_http, tenant_id, timeline_id, last_flush_lsn)
     return last_flush_lsn
+
+
+def parse_project_git_version_output(s: str) -> str:
+    """
+    Parses the git commit hash out of the --version output supported at least by neon_local.
+
+    The information is generated by utils::project_git_version!
+    """
+    import re
+
+    res = re.search(r"git(-env)?:([0-9a-fA-F]+)(-\S+)?", s)
```

Comment:
```suggestion
    res = re.search(r"git(-env)?:([0-9a-f]{8,40})(-\S+)?", s, re.IGNORECASE)
```

Reply:
I'd prefer not to do case insensitive, fine with the length limitation.
```suggestion
    res = re.search(r"git(-env)?:([0-9a-fA-F]{8,40})(-\S+)?", s)
```